projects/cloud/app/frontend/components/pages/new-project/NewProjectPage.tsx

```diff
@@ -49,17 +49,28 @@ export const NewProjectPage = observer(() => {
               <TextField
                 type="text"
                 label="Organization name"
+                error={
+                  newProjectPageStore.isOrganizationNameValid
+                    ? false
+                    : 'The name is invalid'
+                }
+                helpText="The name must be in the format of organization-name"
```

Comment:
I think it will be useful to tell what characters are allowed instead? From this message I get a perception that it must be "something-something"

Reply:
Thanks for the suggestion, changed 🙌